staging/cse/windows/kubeletfunc.ps1

```diff
@@ -182,7 +182,18 @@ function Get-KubePackage {
         [Parameter(Mandatory = $true)][string]
         $KubeBinariesSASURL
     )
-    Logs-To-Event -TaskName "AKS.WindowsCSE.DownloadKubletBinaries" -TaskMessage "Start to download kubelet binaries and unzip. KubeBinariesPackageSASURL: $global:KubeBinariesPackageSASURL"
+    $mappingFile = [Io.path]::Combine($global:CacheDir, "private-packages\mapping.json")

```

Comment:
> We can check the build logs to confirm whether it is set correctly.

Have we explored using pester to unit test the powershell in agentbaker?
https://github.com/pester/Pester

This is quite dense in logic, and the consequences of a bug are quite dramatic on the end user experience

Reply:
looks like we do use pester in other parts?
https://github.com/Azure/AgentBaker/blob/master/parts/windows/windowscsehelper.tests.ps1

Reply:
Yes. Will add the ut.

Reply:
Added ut.